.travis.yml

```
@@ -0,0 +1,16 @@
+sudo: false
+language: go
+go:
+  - "1.11.x"
+  - "1.10.x"
+  - "1.9.x"
+  - master
```

Is it really needed to test against Go master? We were doing it in the CLI and it slows down builds from quite a lot, as they have to build Go from source.

---

So we ignore errors for it. But TBH i didn’t thought about the build time. 

I would be ok with dropping it. Maybe we could consider testing master against it later on

---

I'd rather leave it off, Travis builds would just take 1-2 minutes without it. And we only do common usage of the Go http package.

---

Do we need all versions?

---

will drop master. Not sure if we really need 1.9 but I think the additional time would be fairly low